exporter/trace/cloudtrace.go

```diff
@@ -256,20 +262,21 @@ func (e *Exporter) Shutdown(ctx context.Context) error {
 	return e.traceExporter.Shutdown(ctx)
 }
 
-func (e *Exporter) sdWithDefaultTraceAttributes(sd *export.SpanSnapshot) *export.SpanSnapshot {
-	newSD := *sd
-	for k, v := range e.traceExporter.o.DefaultTraceAttributes {
+func createKeyValueAttributes(attr map[string]interface{}) []attribute.KeyValue {
+	kv := make([]attribute.KeyValue, len(attr))
```

Comment:
It looks like you are trying to reserve capacity in the slice, not create a slice with several zero elements.

```suggestion
	kv := make([]attribute.KeyValue, 0, len(attr))
```

Reply:
True, I missed that. 
Fixed.